solidity/random-beacon/contracts/libraries/Groups.sol

```diff
@@ -65,16 +85,6 @@ library Groups {
     ) internal {
         bytes32 groupPubKeyHash = keccak256(groupPubKey);
 
-        require(
-            self.groupsData[groupPubKeyHash].registrationBlockNumber == 0,
-            "Group with this public key was already registered"
-        );
-
-        require(
-            self.groupsRegistry.length <= type(uint64).max,
-            "Max number of registered groups reached"
-        );
```

Comment:
I sacrificed this check to not run into contract size limit problems. Even if we were adding one new group per second, [2^64 is more than the age of Earth](https://www.wolframalpha.com/input?i=2**64+seconds+to+years).